.circleci/config.yml

```diff
@@ -35,7 +30,7 @@ jobs:
 
   deploy:
     docker:
-      - image: cimg/node:17.9
+      - image: cimg/node:current
```

Comment:
do we need to upgrade this image? It's only used for semantic release AFAICT

Reply:
weak opinion: maybe we pin the s.r. version when/if it becomes a problem? Seems more reliable to have all action running on `current` than on a pinned version.

Reply:
I'm down for either, just thought I'd throw it out there